pkg/kubelet/cm/cpumanager/state/state_checkpoint_test.go

```diff
@@ -358,11 +358,18 @@ func TestCheckpointStateHelpers(t *testing.T) {
 					if cpus, _ := state.GetCPUSet(pod, container); !cpus.Equals(set) {
 						t.Fatalf("state inconsistent, got %q instead of %q", set, cpus)
 					}
+					if cpus := state.GetCPUSetOrDefault(pod, container); !cpus.Equals(set) {
+						t.Fatalf("state inconsistent, got %q instead of %q", set, cpus)
+					}
```

Comment:
if the intent is to cover `GetCPUSDetOrDefault` I'd rather add an explicit test focused on this helper(s)

Reply:
resolved in 512ca4b9888c303b1b0aef5469293a099bc4cd9b